source/developers-guide/backend-extension/index.md

```diff
@@ -433,5 +437,39 @@ Ext.define('Shopware.apps.SwagExtendCustomer.view.detail.Window', {
 //{/block}
 ```
 
+#### Add custom ExtJS controller
+
+To register a custom controller in an existing backend ExtJS module, first you need to add the file to the app.js:
+
+```js
+//{block name="backend/customer/application"}
+//      {$smarty.block.parent}
+//      {include file="backend/swag_extend_customer/controller/my_own_controller.js"}
+//      {include file="backend/swag_extend_customer/view/detail/my_own_tab.js"}
+//{/block}
+```
+
+After adding it to the app.js file, the fill will be loaded, but the controller will not be instanced.
+
+To instantiate your controller, you need to override an the existing controller (depends on the module you want to extend). In this file you can override existing functions of the parent controller. You can and mostly need to call the parent function with the line `me.callParent(arguments)`:
```

Comment:
... override the existing...

Reply:
"the fill" should be "the file", shouldn't it?

Reply:
Fixed!